### DISCUSSION TITLE
🧩 FEATURE REQUEST: Ability to Add different Social Media Handles

### DISCUSSION THREAD

**Discussion by DionneNoellaBarretto** [dailydotdev/daily#1006] [Feature Requests]
### Description

 I would like to have the ability to add multiple social media handles to my Profile / Account Details - besides the default twitter / github / hashnode / linkedin hotlinks ( also for the profile website link may be only showing a clickable icon like the github/hashnode icon will look better as opposed to filling the space with a truncated url..
 
![image](https://user-images.githubusercontent.com/16610213/142768879-d8c4b632-0de7-4c09-9743-f601aa9de07e.png)

### Solution

Would be nice to allow the user upload an img for their handle as well from a repository of logos that can be crowdsourced based on different handles on different profiles users choose to include on their daily dev profile..

Example: 
![image](https://user-images.githubusercontent.com/16610213/142768819-56c3d6c0-6b35-42fc-9403-5c641b47a60e.png)

![image](https://user-images.githubusercontent.com/16610213/142768798-007ed84b-6d7e-42dc-8788-51f13cc261cd.png)

**Comment by idoshamun** [comment]
Hi @DionneNoellaBarretto,

Thanks for the suggestion, I'll forward it to the team and get back to you on this.

**Comment by rebelchris** [comment]
This has since been added
